app/views/smart_proxies/welcome.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for(:title, _("Smart Proxies")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("sitemap", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Proxies') %></h1>
+  <p><%= _("The Smart Proxy provides an easy way to add or extended existing subsystems, via
```

Comment:
I think this should be rephrased, something like "Smart proy provides restful API to subsystems such as DNS, DHCP, TFTP that Foreman talks to"